### PR TITLE
Update phase 5 and 6 rubrics to match each other

### LINKED ISSUE #156: Update Phase 5 & 6 Specs about Grading & Extra Credit
## Worklist
* Indicate to no partial passoffs are allowed
* softwareconstruction240/autograder#467
* Consolidate the Grading Rubric in Canvas and in the GitHub page (Phase 6 only)
    * #170
* Indicate that extra credit is offered on **Phase 5**, only if they have completed the entire project before the pass off day
    * We're only giving students credit for "the part that takes our time," and only after they've done everything else
* Clearly state that students mush have a **passing submission** on the autograder before doing any manual passoffs with a student.
    * All (17) of the test cases must pass 
    * All quality checks must be passing
    * All manual passoffs must be passing

### PR DESCRIPTION
**Pull request by ThanGerlek**
Specs changes:
- For both P5 and P6, extra credit is given if and only if everything is completely done (including 100% on autograder) BEFORE the due date. Amount is 4% of the functionality score (4pts for P5, 3pts for P6).
- P6 passoff is no longer has partial credit
- P6 unit tests are no longer partial credit
- Combined P6 manual passoff items into one Functionality item

Closes #156

### CODE REVIEW

File: chess/6-gameplay/gameplay.md [hunk 1]
```diff
@@ -197,26 +197,23 @@ For this phase the TAs will grade the quality of your project's source code. The
...
+
+| Category                      |             Criteria                                                                                                                                                                                      |       Points |
+| ----------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -----------: |
+| GitHub History                | At least 12 GitHub commits evenly spread over the assignment period that demonstrate proof of work.                                                                                                       | Prerequisite |
+| Test Cases                    | All passoff and unit tests pass.                                                                                                                                                                          |           50 |
```
Comment (19mdavenport) [inline chess/6-gameplay/gameplay.md:213]: There isn't a unit testing requirement for phase 6, is there?

Comment (ThanGerlek) [reply chess/6-gameplay/gameplay.md:213]: I meant all current and past tests, but you're right, that's a little confusing. Clarified.

File: chess/5-pregame/pregame.md [hunk 2]
```diff
@@ -151,13 +151,19 @@ For this phase the TAs will grade the quality of your project's source code. The
 
 All of the tests in your project must succeed in order to complete this phase.
 
-To pass off this assignment submit your work to the course [auto-grading](https://cs240.click/) tool. When that is done, meet with a TA and demonstrate that your Chess client and server meet all requirements and assign you a final grade.
+To pass off this assignment submit your work to the course [auto-grading](https://cs240.click/) tool. When that is done, meet with a TA and demonstrate that your Chess client and server meet all requirements and they will assign you a final grade.
```
Comment (leesjensen) [inline chess/5-pregame/pregame.md:154]: This seems like it is saying the same thing twice. Just with an exclamation point the second time.

Could it just say:

To pass off this assignment submit your work to the course [auto-grading](https://cs240.click/) tool. Once it completely passes the auto-grader, meet with a TA and demonstrate that your Chess client and server meet all requirements and they will assign you a final grade.

File: chess/5-pregame/pregame.md [hunk 3]
```diff
@@ -151,13 +151,19 @@ For this phase the TAs will grade the quality of your project's source code. The
...
+> [!IMPORTANT]
+> You must completely pass the autograder first, including fixing any code style issues, before the TAs will grade your work.
 
 ### Grading Rubric
 
 > [!IMPORTANT]
 > You are required to commit to GitHub with every minor milestone. For example, after you successfully pass a test. This should result in a commit history that clearly details your work on this phase. If your Git history does not demonstrate your efforts then your submission may be rejected.
 
+> [!NOTE]
+> You can receive 4 points of extra credit by completing this phase early. This means receiving 100% from the autograder, then completing an in-person passoff, before the day it is due.
```
Comment (leesjensen) [inline chess/5-pregame/pregame.md:165]: Perhaps:

> You can receive 4 points of extra credit by first receiving 100% from the autograder and then completing the in-person passoff before the final due date.

**Review by leesjensen — CHANGES_REQUESTED** [review]

**Pull request closed without merging**